### PR TITLE
NOJIRA: Make Manage orders test less failure prone (now retryable)

### PR DESCRIPTION
**Pull request by samuelzerozero**

### CODE REVIEW

File: e2e/tests/adminManagesOrders_test.js [hunk 1]
```diff
@@ -81,12 +83,15 @@ Scenario('Create 32b discharge of care order', async ({ I, caseViewPage, manageO
...
     orderIndex: 1,
     orderType: 'Discharge of care order (C32B)',
-    approvalDate: today,
+    approvalDate: approvalDate,
   });
 });
 
 Scenario('Create EPO order', async ({ I, caseViewPage, manageOrdersEventPage }) => {
-  await setupScenario(I, caseViewPage);
+  const newCaseId = await I.submitNewCaseWithData(caseData);
```
Comment (lewisbirks) [inline e2e/tests/adminManagesOrders_test.js:91]: What is the reason for submitting a new case here?

File: e2e/tests/adminManagesOrders_test.js [hunk 2]
```diff
@@ -118,7 +123,10 @@ Scenario('Create EPO order', async ({ I, caseViewPage, manageOrdersEventPage })
 });
 
 Scenario('Create EPO Prevent removal order', async ({ I, caseViewPage, manageOrdersEventPage }) => {
-  await setupScenario(I, caseViewPage);
+  const newCaseId = await I.submitNewCaseWithData(caseData);
```
Comment (lewisbirks) [inline e2e/tests/adminManagesOrders_test.js:126]: what is the reason for submitting a new case here?

File: e2e/tests/adminManagesOrders_test.js [hunk 3]
```diff
@@ -81,12 +83,15 @@ Scenario('Create 32b discharge of care order', async ({ I, caseViewPage, manageO
   assertOrder(I, caseViewPage, {
     orderIndex: 1,
     orderType: 'Discharge of care order (C32B)',
-    approvalDate: today,
+    approvalDate: approvalDate,
   });
 });
 
 Scenario('Create EPO order', async ({ I, caseViewPage, manageOrdersEventPage }) => {
```
Comment (lewisbirks) [inline e2e/tests/adminManagesOrders_test.js:90]: this one still uses today as the approval date time, want to change that to approvalDate as well?

File: e2e/tests/adminManagesOrders_test.js [hunk 4]
```diff
@@ -118,7 +123,10 @@ Scenario('Create EPO order', async ({ I, caseViewPage, manageOrdersEventPage })
 });
 
 Scenario('Create EPO Prevent removal order', async ({ I, caseViewPage, manageOrdersEventPage }) => {
```
Comment (lewisbirks) [inline e2e/tests/adminManagesOrders_test.js:125]: this one still uses today as the approval date time, want to change that to approvalDate as well?

Comment (samuelzerozero) [reply e2e/tests/adminManagesOrders_test.js:126]: For both EPO had to keep the same dates, as this is constraint to the present and 8 days margin, so I can't use the sequence. In this way I put it in their own case, so don't need too much magic

File: e2e/tests/adminManagesOrders_test.js (hunk 3 above)
Comment (samuelzerozero) [reply e2e/tests/adminManagesOrders_test.js:91]: For both EPO had to keep the same dates, as this is constraint to the present and 8 days margin, so I can't use the sequence. In this way I put it in their own case, so don't need too much magic

File: e2e/tests/adminManagesOrders_test.js (hunk 2 above)
Comment (samuelzerozero) [reply e2e/tests/adminManagesOrders_test.js:127]: For both EPO had to keep the same dates, as this is constraint to the present and 8 days margin, so I can't use the sequence. In this way I put it in their own case, so don't need too much magic

File: e2e/tests/adminManagesOrders_test.js (hunk 1 above)
Comment (samuelzerozero) [reply e2e/tests/adminManagesOrders_test.js:92]: For both EPO had to keep the same dates, as this is constraint to the present and 8 days margin, so I can't use the sequence. In this way I put it in their own case, so don't need too much magic

**Pull request merged**